python/lib/sift_py/file_attachment/__init__.py

```diff
@@ -0,0 +1,88 @@
+"""
+This module contains services to facilitate uploading and downloading file attachments.
+It also provides utilities to easily query all file attachments for a given entity
+which could be a run, annotation, or annotation logs. File attachment deletion is also supported.
+
+Once files have been attached, they should be viewable on the Sift application, attached to their
+respective entities. Below are various examples on how to leverage the `sift_py.file_attachment.service.FileAttachmentService`.
+
+## Initializing the file attachment service
+
+Unlike other services throughout `sift_py`, the `sift_py.file_attachment.service.FileAttachmentService` does rely on both
+REST and gRPC APIs, so with that in mind we can initialize our service like so:
+
+```python
+from sift_py.grpc.transport import SiftChannelConfig, use_sift_channel
+from sift_py.file_attachment.service import FileAttachmentService
+from sift_py.file_attachment.entity import Entity, EntityType
+from sift_py.file_attachment.metadata import ImageMetadata
+from sift_py.rest import SiftRestConfig
+
+from sift.remote_files.v1.remote_files_pb2 import GetRemoteFileRequest
+from sift.remote_files.v1.remote_files_pb2_grpc import RemoteFileServiceStub
+
+rest_config: SiftRestConfig = {
+    # Be sure to exclude the "https://" or "http://" scheme out of the uri
```

Comment:
Mainly curious. Any reason we don't just give the prod / gov examples for the rest endpoints? If it is a security or privacy thing? I am also fine with it as is

Reply:
I fixed the typo that's still in the screenshot. Screenshot is out of date. And yeah it's for security purposes. Something we only want to expose via documentation to customers. It's a minor inconvenience though for sure.